doc/advanced_configuration.rst

```diff
@@ -71,7 +71,7 @@ starting with ``plot_long_examples_``, you would do:
 .. code-block:: python
 
     sphinx_gallery_conf = {
-        'filename_pattern' : '/plot_(?!long_examples)'
+        'filename_pattern' : '^((?!plot_long_examples_).)*$'
```

Comment:
Looking at this old PR, it would actually make sense to have a simpler example along these lines indeed, i.e. just show a regex that skips example starting with `skip`. This is a more common use case that running the examples which start with `plot_` but not the ones starting with `plot_long_examples`. On top of that the doc by saying "For example, to skip building examples starting with ``plot_long_examples_``" is slightly misleading.

I would just modify the documentation. I would neither change conf.py nor add an additional examples that is skipped (we have one already: just_code.py).